test/unit/kernel/src/KernelLambda.cpp

```diff
@@ -73,17 +73,10 @@ struct TestTemplateCapture
 
         alpaka::test::KernelExecutionFixture<TAcc> fixture(alpaka::Vec<Dim, Idx>::ones());
 
-        std::uint32_t const arg = 42u;
+        std::uint32_t arg = 42u;
```

Comment:
Why is removing `const` necessary?

Reply:
I am surprised that this worked. I also don't think it is an elegant solution, because the `const` is rightfully there. But then, I also don't care so much, since it has no effect on the test :)